src/middleware/verifyAuthToken.js

```diff
@@ -1,27 +1,94 @@
 import jwt from 'jsonwebtoken';
 import dotenv from 'dotenv';
+import models from '../database/models';
 
 dotenv.config();
 
 const verifyAdminToken = (req, res, next) => {
   const authHeader = req.headers.authorization;
   if (typeof authHeader === 'undefined') {
-    res.status(401).json({ status: 401, message: 'Unauthorised - Header Not Set' });
+    res.status(401).json({
+      status: 401,
+      message: 'Unauthorised - Header Not Set'
```

Comment:
Good Job @Fiston but can you use message which is familiar to users, I think user can't know the meaning of 'Unauthorised - Header Not Set'

Reply:
thanks isaac, i am going to change that.